lib/shared/sdk/usbboot/index.js

```diff
@@ -284,10 +291,11 @@ class USBBootAdapter extends EventEmitter {
     }, {
       concurrency: 5
     }).catch((error) => {
-      callback(error)
+      this.emit('error', error)
+      callback && callback(error)
```

Comment:
Should we document that using a callback means that you won't get notified of preparation errors? Maybe better to clear the confusion by just supporting events?

Reply:
Yeah, good point – let me do that in a follow-up PR.